viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+movie_dict = {} 
+user_data = {}
+#user_data = {"watched":[{movie},{},{}...],
+#              "watchlist":[{movie},{},{}...],
+#              "friends":[{friend},{},{}...],
+#              "subscriptions": ["string","",""]
+#              "favorites": [{movie},{},{}...]}
+#{movie} --> {"title":"string" , "genre":"string" , "rating":float}
+#{friend} --> {"watched": [{movie2},{},{}...]}
+#{movie2} --> {"title":"string" , "host": "streaming"}
+
+
+#Wave 1 
+#Q1 - add a new movie(title,genre,rating) to movie_dict
+def create_movie(title,genre,rating):
+    if not title or not genre or not rating:
+        return None
+    else:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+#Q2 - add a movie to list "watched"
+def add_to_watched(user_data,movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+#Q3 - add a movie to list "watchlist"
+def add_to_watchlist(user_data,movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#Q4 - move a movie from "watchlist" to "watched" if in watched list, otherwisde return user_data
+def watch_movie(user_data,title):
+    watchlist = user_data["watchlist"]
+    watched = user_data["watched"]
+    for movie in watchlist:
+        if title in movie["title"]:
+            watchlist.remove(movie)
+            watched.append(movie)
```

Comment:
Consider using `add_to_watched`
```suggestion
    for movie in watchlist:
        if title in movie["title"]:
            user_data['watchlist'].remove(movie)
            add_to_watched(user_data, movie)
```